### PR TITLE
refactor(ApplicationRef): Move ApplicationRef to its own file

### PR DESCRIPTION
**Pull request by jteplitz**

### CODE REVIEW

File: modules/angular2/src/web-workers/worker/application_common.ts [hunk 1]
```diff
@@ -147,7 +146,7 @@ export function bootstrapWebWorkerCommon(
     componentInjectableBindings: List<Type | Binding | List<any>> = null): Promise<ApplicationRef> {
   var bootstrapProcess: PromiseCompleter<any> = PromiseWrapper.completer();
 
-  var zone = createNgZone();
+  var zone = new NgZone({enableLongStackTrace: assertionsEnabled()});
```
Comment (yjbanov) [inline modules/angular2/src/web-workers/worker/application_common.ts]: This change seems unrelated. Did you mean to keep it?

Comment (jteplitz602) [inline modules/angular2/src/web-workers/worker/application_common.ts]: Yeah, because we don't want to load core/application_common at all in this file since it loads the DOM. I could also hold off on this change for the full server message bus PR though.
